tests/x11/reboot_gnome.pm

```diff
@@ -26,6 +26,13 @@ sub run() {
         assert_and_click 'reboot-auth-typed', 'right';    # Extra assert_and_click (with right click) to check the correct number of characters is typed and open up the 'show text' option
         assert_and_click 'reboot-auth-showtext';          # Click the 'Show Text' Option to enable the display of the typed text
         assert_screen 'reboot-auth-correct-password';     # Check the password is correct
+
+        # we need to kill ssh for iucvconn here,
+        # because after pressing return, the system is down
+        if (check_var('BACKEND', 's390x')) {
+            console('iucvconn')->kill_ssh;
```

Comment:
I suggest to map these lines into utility function so you can extend later for other backends. Something along "prepare_for_system_shutdown"